src/components/molecules/Common/AssetModal/AssetListItem/index.tsx

```diff
@@ -25,12 +25,18 @@ export type Props = {
   onCheck?: (checked: boolean) => void;
 };
 
-const AssetListItem: React.FC<Props> = ({ asset, selected, checked, isImage, onCheck }) => {
+const AssetListItem: React.FC<Props> = ({ asset, selected, checked, onCheck }) => {
   const theme = useTheme();
   return (
     <ListItem key={asset.id} align="center" selected={selected} onClick={() => onCheck?.(!check)}>
       <Icon
-        icon={checked ? "checkCircle" : isImage ? "image" : "file"}
+        icon={
+          checked
+            ? "checkCircle"
+            : /\.(jpg|jpeg|png|gif|GIF|JPG|PNG)$/.test(asset.url)
```

Comment:
Here needs to be updated like in the AssetCard. I am sure isImage isn't the greatest, but I am not sure the same regex in multiple areas is good either.
Any big reason you removed the isImage?

Reply:
yes, sure. I m not update here yet. thanks

Reply:
yes , why I remove is because the Assets container in the setting page, there are not only image type, and also kml type and others. and also the image type should show the preview image, but the kml file don't need. So the card should decide by themselves, not get the isImage from outside. So I think this way is better @KaWaite

Reply:
Since it seems we are keeping it this way, double and triple check you've deleted all reference to isImage